pom.xml

```diff
@@ -110,6 +110,7 @@
     <javac.target>1.8</javac.target>
     <redirectTestOutputToFile>true</redirectTestOutputToFile>
     <testRetryCount>2</testRetryCount>
+    <src.dir>src/main/java</src.dir>
```

Comment:
if this is only for "delombok" then what about giving a more specific name ? like output.delombok.dir ?

Reply:
it's also used by javadoc maven plugin.